node/state.re

```diff
@@ -270,7 +282,10 @@ let load_snapshot =
       block_height,
       last_block_hash,
       state_root_hash,
-      last_state_root_update: 0.0,
+      // The node will only sign blocks that have an
+      // acceptable state root hash relative to the last_state_root_update.
+      // see [building_blocks.re].
+      last_state_root_update: Unix.time(),
```

Comment:
This should not be Unix.time, as otherwise the node may sign a block that contains an old state_root_hash even if it's already too late.